compiler/esy.json

```diff
@@ -15,12 +15,12 @@
   "scripts": {
     "copy-compiler": "cp #{$GRAINC_BIN_PATH} #{$CLI_BIN_DIR}",
     "clean": "rm -rf #{self.root}/_esy",
-    "test": "dune runtest --force",
+    "test": "dune runtest",
```

Comment:
Any way we can keep `--force`? It's useful for when you change the runtime and want to re-run the tests.

Reply:
I didn't know what this did 🙈 Re-added